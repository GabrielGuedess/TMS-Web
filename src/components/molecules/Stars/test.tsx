import { render } from '@testing-library/react';

import { Stars } from '.';

jest.mock('@react-three/fiber', () => ({
  useFrame: () => ({}),
  Canvas: () => <div data-testid="Mock Canvas" />,
}));

jest.mock('@react-three/drei', () => ({
  Points: () => <div data-testid="Mock Points" />,
  PointMaterial: () => <div data-testid="Mock PointMaterial" />,
}));

describe('<Stars />', () => {
  it('should render the stars correctly', () => {
    const { container } = render(<Stars />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
