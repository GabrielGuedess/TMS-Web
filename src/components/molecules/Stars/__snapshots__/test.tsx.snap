// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Stars /> should render the stars correctly 1`] = `
<div
  data-testid="Mock Canvas"
/>
`;
